enocean/__init__.py

```diff
@@ -441,7 +441,7 @@ def start_UTE_learnmode(self, id_offset=0):
 
     def _send_UTE_response(self, data, optional):
         choice = data[0]
-        #payload = data[1:-5]
+        payload = data[1:-5]
```

Comment:
This looks like an important fix, since the `payload` variable is use later. Without this fix the plugin method failed due to undefined variable, right? Was it ever woriking?

Reply:
I doubt that it was working before. At least in my case, it wasn't.  After removing the hash signed at least it did work..